ksql-parser/src/main/java/io/confluent/ksql/parser/tree/CreateStream.java

```diff
@@ -28,22 +28,12 @@ public CreateStream(
       final QualifiedName name,
       final TableElements elements,
       final boolean notExists,
-      final Map<String, Literal> properties
+      final CreateSourceProperties properties
   ) {
     this(Optional.empty(), name, elements, notExists, properties);
   }
 
   public CreateStream(
-      final Optional<NodeLocation> location,
```

Comment:
I replaced the `Map<String, Literal>` here with `CreateSourceProperties` as its only used in tests and should of been changed when the production change was made.